server.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
```

Comment:
Btw, no one uses `use strict` in node

Reply:
Also, let's rename this file to `app.js` to adhere to conventions

Reply:
Curious, why do we not do `use strict` in node? Doesn't node use v8?

Reply:
I honestly have no idea why people in node don't use it. It does work, I just don't know if people just rely on linters.